<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelisationPrblm</class>
 <widget class="QMainWindow" name="ModelisationPrblm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Acceuil</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>Notebad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Aide</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Lancer la simulation</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QFormLayout" name="formLayout_8">
          <item row="4" column="0">
           <widget class="QLabel" name="label_chaineE">
            <property name="text">
             <string>Chaine_evaluation</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>a+b</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_intervalleMI">
            <property name="text">
             <string>Intervalle minimun</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_intervalleMA">
            <property name="text">
             <string>Intervalle maximum</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_generationS">
            <property name="text">
             <string>Generation satisfaisante</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="text">
             <string>150</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Clavier Visuel</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="pushButton_14">
            <property name="text">
             <string>Clavier Visuel</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="10" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="12" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="13" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Sauvgarder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Charger</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_8"/>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="3" column="0">
               <widget class="QLabel" name="label_nombreG">
                <property name="text">
                 <string>Nombre de genes</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="spinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>8</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_tailleP">
                <property name="text">
                 <string>Taille de population</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="spinBox_2">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>2048</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_nombreI">
                <property name="text">
                 <string>Nombre d'iterarions</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSpinBox" name="spinBox_3">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_nombreIS">
                <property name="text">
                 <string>Nombre individu a selectiones</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QSpinBox" name="spinBox_4">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>1500</number>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_tauxC">
                <property name="text">
                 <string>Taux de croisement</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_tauxM">
                <property name="text">
                 <string>Taux de mutation</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_typeS">
                <property name="text">
                 <string>Type de selection</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Selection 1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Selection 2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Selection 3</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_typeG">
                <property name="text">
                 <string>Type_genes</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QLabel" name="label_senseE">
                <property name="text">
                 <string>Le sense de l'evaluation</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <widget class="QRadioButton" name="radioButton_2">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_3">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_14"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.050000000000000</double>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <widget class="QRadioButton" name="radioButton_5">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_4">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_10"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_11"/>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Nom fichier latex</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_6">
              <property name="text">
               <string>fichierLatx</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>Generer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Reconfigurer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Reprondre</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Nom de fichier sauvgarde</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>Ok</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Nom de fichier a charger</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string>ok</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
